### ISSUE TITLE
stringTest=long recursion

### ISSUE THREAD

**Issue by phet-steele** [phetsims/pendulum-lab#188]
Discovered by @BryceAG, the following link using `?stringTest=long` will not load if:

- run on Windows AND Firefox.

http://www.colorado.edu/physics/phet/dev/html/pendulum-lab/1.0.0-rc.2/pendulum-lab_en.html?stringTest=long

Prints this warning:

```
too much recursion phetAllocation.js:43:1
	phetAllocation http://phettest.colorado.edu/phet-core/js/phetAllocation.js:43:1
	Vector2 http://phettest.colorado.edu/dot/js/Vector2.js:38:23
	minus http://phettest.colorado.edu/dot/js/Vector2.js:390:14
	getStartTangent http://phettest.colorado.edu/kite/js/segments/Line.js:150:25
	get startTangent http://phettest.colorado.edu/kite/js/segments/Line.js:154:33
	addSegmentDirectly http://phettest.colorado.edu/kite/js/util/Subpath.js:132:7
	addSegment http://phettest.colorado.edu/kite/js/util/Subpath.js:153:9
	addSegmentAndBounds http://phettest.colorado.edu/kite/js/Shape.js:1436:7
	arcPoint http://phettest.colorado.edu/kite/js/Shape.js:483:9
	arc http://phettest.colorado.edu/kite/js/Shape.js:461:93
	Shape.roundedRectangleWithRadii http://phettest.colorado.edu/kite/js/Shape.js:1654:7
	getTitleBarShape http://phettest.colorado.edu/sun/js/AccordionBox.js:453:14
	layout http://phettest.colorado.edu/sun/js/AccordionBox.js:375:37
	layout self-hosted:941:17
	trigger0 http://phettest.colorado.edu/axon/js/Events.js:270:9
	validateBounds http://phettest.colorado.edu/scenery/js/nodes/Node.js:1100:11
	validateBounds http://phettest.colorado.edu/scenery/js/nodes/Node.js:1108:9
	validateBounds http://phettest.colorado.edu/scenery/js/nodes/Node.js:1108:9
	validateBounds http://phettest.colorado.edu/scenery/js/nodes/Node.js:1108:9
	validateBounds http://phettest.colorado.edu/scenery/js/nodes/Node.js:1108:9
	validateBounds http://phettest.colorado.edu/scenery/js/nodes/Node.js:1108:9
	validateBounds http://phettest.colorado.edu/scenery/js/nodes/Node.js:1108:9
	...
```

Seen on Windows 7, 8.1, and 10 FF. For phetsims/QA/issues/45.

**Comment by jonathanolson** [comment]
General workaround/fix applied to Scenery. Added scenery branch (and included in 1.0 branch).

**Comment by brroberts1231** [comment]
Tested on Win 7, 8, and 10. Fixed on all, closing.
